### PR TITLE
Add descriptions and units of constant variables

### PR DESCRIPTION
**Pull request by smileMchen**
TYPE: no impact, text only

KEYWORDS: constant variables

SOURCE: internal

DESCRIPTION OF CHANGES: descriptions and units of variables listed in share/module_model_constants.F are added to meet the requirement of CCPP 

LIST OF MODIFIED FILES: 
M       share/module_model_constants.F

TESTS CONDUCTED:  not necessary

### CODE REVIEW

**Comment by smileMchen** [comment]
@davegill @weiwangncar @dudhia @kkeene44 
There are some variables that are combinations of other constants. I am not sure how to describe them. Please let me know if you have any suggestions.

**Comment by davegill** [comment]
@llpcarson 
Laurie,
We are going through these constants to help us out when we start using them in the shared physics. Please take a look at the constants that are blank AND likely from NMM schemes.

File: share/module_model_constants.F [hunk 1]
```diff
@@ -17,81 +17,81 @@ MODULE module_model_constants
...
-
-       REAL , PARAMETER ::  pq0=379.90516
-       REAL , PARAMETER ::  epsq2=0.2
+   REAL    , PARAMETER ::  SVP1=0.6112      ! constant for saturation vapor pressure (dimensionless)
+   REAL    , PARAMETER ::  SVP2=17.67       ! constant for saturation vapor pressure (dimensionless)
+   REAL    , PARAMETER ::  SVP3=29.65       ! constant for saturation vapor pressure (K)
+   REAL    , PARAMETER ::  SVPT0=273.15     ! constant for saturation vapor pressure (K)
+   REAL    , PARAMETER ::  EP_1=R_v/R_d-1.  !  constant for virtual temperature (r_v/r_d - 1) (dimensionless)
+   REAL    , PARAMETER ::  EP_2=R_d/R_v     ! constant for specific humidity calculation (dimensionless)
+   REAL    , PARAMETER ::  KARMAN=0.4               ! Karman constant
```
Comment (weiwangncar) [inline share/module_model_constants.F]: Probably 'von Karman constant'?

File: share/module_model_constants.F [hunk 2]
```diff
@@ -17,81 +17,81 @@ MODULE module_model_constants
...
-   REAL    , PARAMETER ::  STBOLT=5.67051E-8
-
-   REAL    , PARAMETER ::  prandtl = 1./3.0
-                                         ! constants for w-damping option
-   REAL    , PARAMETER ::  w_alpha = 0.3 ! strength m/s/s
-   REAL    , PARAMETER ::  w_beta  = 1.0 ! activation cfl number
-
-       REAL , PARAMETER ::  pq0=379.90516
-       REAL , PARAMETER ::  epsq2=0.2
+   REAL    , PARAMETER ::  SVP1=0.6112      ! constant for saturation vapor pressure (dimensionless)
```
Comment (weiwangncar) [inline share/module_model_constants.F]: For the following three, it is probably better to say 'constant for saturation vapor pressure calculation (dimensionless)'.

File: share/module_model_constants.F [hunk 3]
```diff
@@ -17,81 +17,81 @@ MODULE module_model_constants
...
+   REAL    , PARAMETER ::  SVP2=17.67       ! constant for saturation vapor pressure (dimensionless)
+   REAL    , PARAMETER ::  SVP3=29.65       ! constant for saturation vapor pressure (K)
+   REAL    , PARAMETER ::  SVPT0=273.15     ! constant for saturation vapor pressure (K)
+   REAL    , PARAMETER ::  EP_1=R_v/R_d-1.  !  constant for virtual temperature (r_v/r_d - 1) (dimensionless)
+   REAL    , PARAMETER ::  EP_2=R_d/R_v     ! constant for specific humidity calculation (dimensionless)
+   REAL    , PARAMETER ::  KARMAN=0.4               ! Karman constant
+   REAL    , PARAMETER ::  EOMEG=7.2921E-5          ! angular velocity of rotation (rad^-1)
```
Comment (weiwangncar) [inline share/module_model_constants.F]: Probably better to say 'angular velocity of earth's rotation'?

File: share/module_model_constants.F [hunk 4]
```diff
@@ -17,81 +17,81 @@ MODULE module_model_constants
...
 
 #if ( NMM_CORE == 1 )
-   REAL    , PARAMETER :: r_d          = 287.04
-   REAL    , PARAMETER :: cp           = 1004.6
+   REAL    , PARAMETER :: r_d          = 287.04     ! gas constant for dry air (J deg^-1 kg^-1)
+   REAL    , PARAMETER :: cp           = 1004.6     ! Specific heat of dry air at constant pressure (J deg^-1 kg^-1)
 #else
-   REAL    , PARAMETER :: r_d          = 287.
-   REAL    , PARAMETER :: cp           = 7.*r_d/2.
+   REAL    , PARAMETER :: r_d          = 287.       ! gas constant (J deg^-1 kg^-1)
```
Comment (weiwangncar) [inline share/module_model_constants.F]: How about 'gas constant of dry air'?

File: share/module_model_constants.F (hunk 3 above)
Comment (smileMchen) [reply share/module_model_constants.F]: @weiwangncar 
Wei,
I have made changes following your suggestions. Thanks.
Ming

**Pull request merged**